src/views/Player.vue

```diff
@@ -1,66 +1,90 @@
 <template>
-  <div class="flex flex-col bg-white w-full h-full overflow-hidden">
-    <div>
+  <div>
+    <splash
+      v-if="!isQuestionShown"
+      :title="title"
+      :subject="metadata.subject"
+      :classNumber="metadata.class"
+      :numQuestions="questions.length"
+      :examType="metadata.examType"
+      @start="startQuiz"
+      data-test="splash"
+    ></splash>
+    <div
+      v-if="isQuestionShown"
+      class="flex flex-col bg-white w-full h-full overflow-hidden"
+    >
       <!-- header -->
-      <item-header @skip-item="skipItem" data-test="header"></item-header>
+      <question-header
+        @skip-question="skipQuestion"
+        data-test="header"
+      ></question-header>
     </div>
   </div>
 </template>
 
 <script lang="ts">
-import ItemHeader from "../components/Items/Header.vue";
-import { defineComponent, reactive, toRefs } from "vue";
-import { Item } from "../types";
+import QuestionHeader from "../components/Questions/Header.vue";
+import Splash from "../components/Splash.vue";
+import { defineComponent, reactive, toRefs, computed } from "vue";
+import { Question } from "../types";
 export default defineComponent({
   name: "Player",
   components: {
-    ItemHeader,
+    QuestionHeader,
+    Splash,
   },
   setup() {
     const state = reactive({
-      currentItemIndex: 0 as number,
-      items: [
+      currentQuestionIndex: -1 as number,
+      title: "Geometry Quiz",
+      metadata: {
+        examType: "CBSE",
+        subject: "Maths",
+        class: 8,
+      },
+      questions: [
         {
-          type: "question",
-          details: {
-            type: "mcq",
-            options: ["", ""],
-            correct_answer: 0,
-            image: null,
-            has_char_limit: false,
-            max_char_limit: null,
-          },
+          type: "mcq",
+          options: ["", ""],
+          correct_answer: 0,
+          image: null,
+          has_char_limit: false,
+          max_char_limit: null,
         },
         {
-          type: "question",
-          details: {
-            type: "subjective",
-            options: null,
-            correct_answer: null,
-            image: null,
-            has_char_limit: true,
-            max_char_limit: 1000,
-          },
+          type: "subjective",
+          options: null,
+          correct_answer: null,
+          image: null,
+          has_char_limit: true,
+          max_char_limit: 1000,
         },
         {
-          type: "question",
-          details: {
-            type: "checkbox",
-            options: ["", "", ""],
-            correct_answer: [0, 1],
-            image:
-              "https://plio-prod-assets.s3.ap-south-1.amazonaws.com/images/afbxudrmbl.png",
-            has_char_limit: false,
-            max_char_limit: null,
-          },
+          type: "checkbox",
```

Comment:
Abhi don't tell me this has something to do with the `SplashScreen`? 😛

Reply:
haan related to the same point about removing the idea of `items` and keeping everything as `questions`.